script/Ciel/Day 4/04_01_CIEL04_4_2.txt

```diff
@@ -1,37 +1,40 @@
 [sha:f21616ab0366d75979f7381dfeca3e5daad62caa]{
 -- Page 94, Offset 8410.
 -- ……いや、いくらそんな目をされても頷けない。
-...No matter how much she looks at me with those eyes, I can't agree.
+No matter how much she looks at me with those eyes, I can't agree.
 }
 [sha:34ec7c37d9746ea6965c4c43adad452fda8f7b8d]{
 -- Page 94, Offset 8411.
 -- バラバラにされても生き返る女だの、黒い豹だの燃える死骸だのといった非常識を、相手にできる筈がない。
-There is no way I could deal with the absurdity of a woman who could come back to life after being torn apart, or a black panther, or burning corpses.
+A woman who can come back to life even after being torn to shreds, black panthers, burning corpses...
```

Comment:
Can we move 'being torn to shreds' to the end of the sentence. 'Black panthers' and 'burning corpses' feel like they don't quite fit in with the current structure.

Reply:
As in the entire "A woman.. torn to shreds" block? Not sure what you mean about the structure here

Reply:
Actually ignore me, I didn't comprehend that 'black panthers' and 'burning corpses' were separate topics. Was still operating with the meaning of the og sentence that the topic of the whole sentence was about arc.